src/components/screens/Overviews.js

```diff
@@ -349,6 +363,15 @@ export const Overviews = ({ navigation, route }) => {
           <ListComponent
             ListHeaderComponent={
               <>
+                <Filter
+                  filterTypes={filterTypes}
+                  initialFilters={queryVariables}
+                  isOverlay
+                  setQueryVariables={setQueryVariables}
+                />
+
+                <Divider />
```

Comment:
i think this should be placed where the filter by opening time is placed, to be not part of the scrollable list but always reachable